ctapipe/visualization/mpl_array.py

```diff
@@ -149,7 +149,7 @@ def set_vector_uv(self, u, v, c=None, **kwargs):
         if self._quiver is None:
             coords = self.tel_coords
             self._quiver = self.axes.quiver(
-                coords.x, coords.y,
```

Comment:
Better use a specific unit here? `to_value(u.m)`?

Or is these really unit agnostiv

Reply:
It should be agnostic, but probably isn't. I've chanted it to meters, since that's assumed elsewhere (e.g. in "length").  The problem seems to have been the UV coordinates, though, so i'm fixing that now. They are in m, but quiver doesn't work with unit quantities.

Reply:
It was driving me crazy debugging this, until I just realized that the variable "u" is overwriting the package `import astropy.units as u`...

Reply:
Fixed it now, and now everywhere assumes or converts to meters.

Reply:
> It was driving me crazy debugging this, until I just realized that the variable "u" is overwriting the package import astropy.units as u.

F**K